geo/src/algorithm/line_measures/metric_spaces/geodesic.rs

```diff
@@ -36,7 +37,13 @@ impl Bearing<f64> for Geodesic {
     /// [geodesic line]: https://en.wikipedia.org/wiki/Geodesics_on_an_ellipsoid
     /// [Karney (2013)]:  https://arxiv.org/pdf/1109.4448.pdf
     fn bearing(origin: Point<f64>, destination: Point<f64>) -> f64 {
```

Comment:
All these implementations were copy/pasted (with some variable renaming) from the deprecated traits.

So, instead of delegating to the deprecated traits, the deprecated traits now delegate to the modern ones.

Reply:
> So, instead of delegating to the deprecated traits, the deprecated traits now delegate to the modern ones.

Oh! The 2 exceptions to this are that the deprecated `GeodesicBearing` and `HaversineBearing` do *not* delegate to the modern traits. They have kept their legacy implementation in tact to preserve the fact that their output range is [-180...180], whereas all the modern Bearing traits have unified around [0...360].

See #1210 for more details.